problems/secondSmallest.js

```diff
@@ -6,8 +6,11 @@
 * @returns {number} - Returns the second smallest number.
 */
 
-function secondSmallest() {
-  
+function secondSmallest(arr = []) {
+    let secondSmallest = arr[0, 1, 2];
```

Comment:
We can only index into an array at one index at a time. This will default to the first element in the array.